### PR TITLE
Update custom_image_saas.md

### PR DESCRIPTION
**Pull request by louis-zepl**

### CODE REVIEW

File: docs/guide/enterprise/custom_image_saas.md [hunk 1]
```diff
@@ -1,3 +1,154 @@
-## Custom Images - SaaS
+# Custom Images
+
+Creating a reproducible environment for data science is a common challenge when working as a team on a data science challenge. It is common practice to put several install commands at the beginning of a notebook or script to help ensure that others will be able to recreate your analysis. Unfortunately, this installation step can take a long time when many different libraries and packages are required.
```
Comment (zshainsky) [inline docs/guide/enterprise/custom_image_saas.md]: Remove '...on a data science challenge.'

File: docs/guide/enterprise/custom_image_saas.md [hunk 2]
```diff
@@ -1,3 +1,154 @@
...
+
+To build your custom image, click the “Create” button at the bottom. Depending on the number of libraries in your image, building the image can take anywhere from 5 to 30 minutes. Sometimes builds can fail if the wrong syntax is used, or there is a library version mismatch issue. If that occurs, you can download a log in the image actions which describes where the build failed.  If the image builds successfully, congrats! You can now attach it to notebooks. If you’d like the image you created to be the standard for your entire organization, you can set it as the default image in the image actions. You can also edit or delete an image under these image actions. 
+
+# Adding Python Interpreter & Libraries
+
+To add a Python interpreter to a Custom Image, simply click “+ Python” in the image creation screen. Zepl supports Python 3.8 as the default Python interpreter for custom images. Today, you cannot edit the %alias of an interpreter nor can you add more than one version of a Python interpreter to a Custom Image.
```
Comment (zshainsky) [inline docs/guide/enterprise/custom_image_saas.md]: Spin this to talk about what we do support:

Change: 'Zepl supports Python 3.8 as the default Python interpreter for custom images.' TO 'Zepl supports Python 3.8 as the default Python interpreter and can be referenced by the `%python` alias.'

Remove: 'Today, you cannot edit the %alias of an interpreter nor can you add more than one version of a Python interpreter to a Custom Image. '

File: docs/guide/enterprise/custom_image_saas.md [hunk 3]
```diff
@@ -1,3 +1,154 @@
...
+To add a Python interpreter to a Custom Image, simply click “+ Python” in the image creation screen. Zepl supports Python 3.8 as the default Python interpreter for custom images. Today, you cannot edit the %alias of an interpreter nor can you add more than one version of a Python interpreter to a Custom Image. 
+
+<img src=“/img/CI_CREATE_NEW_PYTHON.png” class=“image-box img-100”/>
+
+To add libraries to your custom image, you can use the following supported to install libraries using pip:
+
+    Libraryname
+    libraryname==version
+
+In addition to any libraries that you include in your custom image list, Zepl installs multiple Python libraries to the image necessary for Zepl to function well. If you try to install a different version of these libraries, it is possible you may run into a compatibility issue. You can always see what versions of libraries you have installed in your image by running a pip list command in the notebook. As of July 2020, the list of libraries we install for Python in a custom image are the following:
```
Comment (zshainsky) [inline docs/guide/enterprise/custom_image_saas.md]: Would it be easy to add a code block who how to call the pip list command with the interpreter...?

%python
!pip list

File: docs/guide/enterprise/custom_image_saas.md [hunk 4]
```diff
@@ -1,3 +1,154 @@
...
+    mysql-connector-python==8.0.18 
+    snowflake-connector-python==2.2.7 
+    hdbcli==2.4.167 
+    psycopg2-binary==2.8.4 
+    pyodps==0.8.4
+
+
+# Adding R Interpreter & Libraries
+
+To add an R interpreter to a Custom Image, simply click “+ R” in the interpreter creation screen. Zepl supports R 3.6 as the default R interpreter for custom images. Today, you cannot edit the %alias of an interpreter nor can you add more than one version of an R interpreter to a Custom Image.
```
Comment (zshainsky) [inline docs/guide/enterprise/custom_image_saas.md]: Sam as above ... Spin this to talk about what we do support

Change: 'Zepl supports R 3.6 as the default R interpreter for custom images' TO 'Zepl supports R 3.6 as the default R interpreter and can be referenced by the %r or %spark.r alias.'

Remove: 'Today, you cannot edit the %alias of an interpreter nor can you add more than one version of a Python interpreter to a Custom Image. '

File: docs/guide/enterprise/custom_image_saas.md [hunk 5]
```diff
@@ -1,3 +1,154 @@
...
+    DatabaseConnector 
+    dplyr 
+    knitr 
+    rJava 
+    RJDBC 
+    snowflakedb/dplyr-snowflakedb
+
+# Adding Spark Interpreter & Libraries
+
+To add a Spark interpreter to a Custom Image, simply click “+ Spark” in the interpreter creation screen. Zepl supports Spark 2.3.2 as the default Spark interpreter for custom images. Today, you cannot edit the %alias of an interpreter nor can you add more than one version of a Spark interpreter to a Custom Image.
```
Comment (zshainsky) [inline docs/guide/enterprise/custom_image_saas.md]: Sam as above ... Spin this to talk about what we do support

Change: 'Zepl supports Spark 2.3.2 as the default Spark interpreter for custom images' TO 'Zepl supports Spark 2.3.2 as the default Spark interpreter and can be referenced by the %spark alias.'

Remove: 'Today, you cannot edit the %alias of an interpreter nor can you add more than one version of a Python interpreter to a Custom Image. '

**Review by zshainsky — COMMENTED** [review]
Quick edits for Louis

File: docs/guide/enterprise/custom_image_saas.md (hunk 3 above)
Comment (louis-zepl) [reply docs/guide/enterprise/custom_image_saas.md]: Yes will do

File: docs/guide/enterprise/custom_image_saas.md [hunk 6]
```diff
@@ -1,3 +1,154 @@
...
+
+When you are creating a new notebook, you must select the Custom Image you want to use to execute code from that notebook.
+
+<img src=“/img/CI_NB_CREATE.png” class=“image-box img-100”/>
+
+In the notebook, we’ve designed a toolbar on the right hand side which helps you see what interpreters and libraries are available for you to use in your custom image. 
+
+<img src=“/img/CI_NB_SIDEBAR.png” class=“image-box img-100”/>
+
+You can change this selection at any time for a given notebook by editing the notebook settings - to do so, your container must be shut down.
```
Comment (BryanKatis) [inline docs/guide/enterprise/custom_image_saas.md]: settings - to do so, your container must be shut down.  Should be ' settings.  To do so, your container must be shut down.

File: docs/guide/enterprise/custom_image_saas.md [hunk 7]
```diff
@@ -1,3 +1,154 @@
-## Custom Images - SaaS
+# Custom Images
+
+Creating a reproducible environment for data science is a common challenge when working as a team on a data science challenge. It is common practice to put several install commands at the beginning of a notebook or script to help ensure that others will be able to recreate your analysis. Unfortunately, this installation step can take a long time when many different libraries and packages are required. 
+
+This is where Zepl Custom Images can help you and your team save significant time. Every time you run a notebook, you are spinning up a container which contains an “image” - a prebuilt environment with the all the libraries and settings your team needs for reproducible data science. Custom Images lets you create images that contain as many or as few libraries as your team needs.
```
Comment (BryanKatis) [inline docs/guide/enterprise/custom_image_saas.md]: 1. 'an “image” - a prebuilt environment' replaced with 'an “image”, which is a prebuilt environment
2. environment with the all the libraries and settings your team needs for reproducible data science. Custom Images lets you create images that contain as many or as few libraries as your team needs.
This should be 'environment with all the libraries and settings your team needs for reproducible data science. Custom Images lets you create images that contain as many or as few libraries as your team needs.

File: docs/guide/enterprise/custom_image_saas.md (hunk 5 above)
Comment (louis-zepl) [reply docs/guide/enterprise/custom_image_saas.md]: I made this change

File: docs/guide/enterprise/custom_image_saas.md (hunk 7 above)
Comment (louis-zepl) [reply docs/guide/enterprise/custom_image_saas.md]: I made these two changes

File: docs/guide/enterprise/custom_image_saas.md (hunk 6 above)
Comment (louis-zepl) [reply docs/guide/enterprise/custom_image_saas.md]: I made these changes

**Review by BryanKatis — APPROVED** [review]
All good!

**Pull request merged**